lib/MySQL_Monitor.cpp

```diff
@@ -637,6 +750,35 @@ MySQL_Monitor::~MySQL_Monitor() {
 	AWS_Aurora_Hosts_Map.clear();
 };
 
+void MySQL_Monitor::p_update_metrics() {
```

Comment:
For each counter:
* take the counter value
* increment it with the value of another variable

Let's make it simple (less code, easier to read, less error prone) and replace it with a macro or takes two arguments: the counter and the variable

Reply:
Done with a helper inline function.